engineering/workflow/default-workflow.md

```diff
@@ -20,92 +20,123 @@ This section describes how our development team carries out its planning and day
+- Engineering: for technical decision-making
+- Product: for product decision-making
+- Partnerships and Community Success: for community/grant decision-making.
 
 (meetings:sprint-planning)=
-### Sprint planning meeting
+### 2. Iteration Planning
 
-The team conducts a sprint iteration planning meeting for 60 minutes at the beginning of each sprint.
+The team conducts a synchronous iteration planning meeting for 60 minutes at the beginning of each iteration.
 
-The goal of this meeting is to synchronize with one another and commit to work to do during the sprint.
+The goal of this meeting is to synchronize with one another and commit to work to do during the iteration that gets us closer to achieving the initiatives objectives.
 
-As part of this meeting, we look to:
-- Not overwhelm any team member
-- Under-estimate our team's total capacity, to provide room for unexpected work (e.g., support work)
-- Have at least one team member assigned to each committed item on the board
+#### Planning Outcomes:
+At the end of this meeting, we will have:
+1. A list of team deliverables that the team commits to completing within the iteration time box. These deliverables will represent work from:
+    - Grant/Funding
+    - Partnerships
+    - Product Roadmap
+    - Support request
+    - Internal Engineering
+    - Upstream community
 
-See [the Sprint Planning issue template](https://github.com/2i2c-org/team-compass/blob/main/.github/ISSUE_TEMPLATE/meeting-sprint-planning.md) for the agenda / structure of these meetings.
+1. Identified and shared the core risks impacting the deliverables. NB: The meeting will not be used to brainstorm solutions to resolve the risks.
 
-(coordination:sprint-board)=
-### The Sprint Board
+1. Owners assigned to resolve the different risks. These individuals will create working sessions and coordinate the necessary domain experts who will be responsible for resolving the risks. NB: The owner is accountable to ensure the working sessions happen. They are not responsible for the actual solutions as this may belong to separate domain experts. A deliverable cannot be committed as part of the iteration if there are dependencies with other teams AND those other teams have not committed to completing the work within the current iteration or if the work was not previously completed in an earlier iteration.
 
-The [Sprint Board](https://github.com/orgs/2i2c-org/projects/49/views/1) is a place to keep track of the [Deliverables and tasks](coordination:deliverables) our team intends to work on for a two week iteration.
+Tip 🥷: This meetings provides us a golden opportunity to:
+- Not overwhelm any team member
+- Under-estimate our team's total capacity and to provide room for unexpected work (e.g., support work)
+- Limit the amount of work in process across Engineering (system optimisation)
+- Ensure that we are focused on delivering the highest value work (value optimisation).
```

Comment:
````suggestion
```{tip}
This meeting provides us a golden opportunity to:
- Not overwhelm any team member
- Under-estimate our team's total capacity and to provide room for unexpected work (e.g., support work)
- Limit the amount of work in process across Engineering (system optimisation)
- Ensure that we are focused on delivering the highest value work (value optimisation).
```
````